### ISSUE TITLE
Creating and administering a topic about BIT Country at the big crypto forum cryptotalk.org (English)

### ISSUE THREAD

**Issue by lesnikutsa** [bit-country/ambassador#136]
My second topic, but already in English. I plan to contribute as much information as possible about the project every day and not be limited to just the starting post. https://cryptotalk.org/topic/356051-bit-country-nft-metaverse/

Please tell me - I also plan to post on the altcointalks forum, where I have the status of a legendary member. Will there be additional points for this? Thank you https://www.altcoinstalks.com/index.php?action=profile;u=28119

**Comment by Mirannnda** [comment]
> My second topic, but already in English. I plan to contribute as much information as possible about the project every day and not be limited to just the starting post. https://cryptotalk.org/topic/356051-bit-country-nft-metaverse/

Hey, thank you for sharing different contents with the public, we can see how much effort you put in here as an Ambassador Candidate. Definitely, your content doesn't need to limited to blogging,  you can share the format you good at, like an amazing video, an explaining video, a tutorial post,  or an interesting meme e.g. Anyway, do what you good at and you think right for B.C. We have briefly reviewed all the content you created, and we are blown away by your artwork skills and blogging. Keep going,  your score will be released soon.

**Comment by Mirannnda** [comment]
> Please tell me - I also plan to post on the altcointalks forum, where I have the status of a legendary member. Will there be additional points for this? Thank you https://www.altcoinstalks.com/index.php?action=profile;u=28119

Yes, for the same content if you share on different platforms, like Altcointalks, we will give an additional point based on the original content you created.  For example, if the content you share on your personal blog got 8 points, and you re-share it on Altcointalks, we will add 0.5-3 points on this blog. Depends on the platform you share or depends on the engagement and length of the sharing, we will seriously score everything, so no worries.
